sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.PartitionProcessor.cs

```diff
@@ -469,15 +469,13 @@ private static string GetOperationDetails(EventProcessorHostPartition context, s
                 }
 
                 // Log RuntimeInformation if EnableReceiverRuntimeMetric is enabled
-                if (context.LastEnqueuedEventProperties != null)
```

Comment:
[CS8073 - The result of the expression is always 'false' (or 'true')](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/warning-waves?f1url=%3FappId%3Droslyn%26k%3Dk(CS8073)#cs8073---the-result-of-the-expression-is-always-false-or-true)

Reply:
What was the original intent of this line of code?  Is it only "always true" because `LastEnqueuedEventProperties` is not marked as nullable?  Can it be null in some cases?  If so, it should probably be marked as nullable instead of always writing the values.

Reply:
This can never be null in the T2 library.   It was carried over from the T1 implementation and should be safe in this form.